crates/ott-balancer-protocol/src/monolith.rs

```diff
@@ -206,16 +206,19 @@ where
 /// Metadata about a room, according to the Monolith.
 pub struct RoomMetadata {
     pub name: RoomName,
+    #[typeshare(serialized_as = "unknown")]
     pub title: serde_json::Value,
+    #[typeshare(serialized_as = "unknown")]
     pub description: serde_json::Value,
-    #[serde(rename = "isTemporary")]
+    #[typeshare(serialized_as = "unknown")]
     pub is_temporary: serde_json::Value,
     pub visibility: Visibility,
-    #[serde(rename = "queueMode")]
+    #[typeshare(serialized_as = "unknown")]
     pub queue_mode: serde_json::Value,
-    #[serde(rename = "currentSource")]
```

Comment:
these renames need to stay. you can put multiple attributes on a field

Reply:
ah, makes sense. thats why the names changed on the monolith end. will change that in a second.